helpers/helpers.js

```diff
@@ -0,0 +1,6 @@
+module.exports.generateRandomImage = function() {
+  let first = Math.floor(Math.random() * 9)
+  let second = Math.floor(Math.random() * 9)
+  let third = Math.floor(Math.random() * 9)
+  return `${first}${second}${third}`
```

Comment:
Have you thought about appending your file type extension here?  (.jpg, .png, etc).

Reply:
+1 on this one, how are you planning on completing the photo path?